release_notes/Service_Fabric_ReleaseNotes_90.md

```diff
@@ -0,0 +1,153 @@
+| 15. | **Feature** | Enforce a configurable Max value on the InstanceCloseDelayDuration | **Brief Description**: Enforcing a configurable Max value on the InstanceCloseDelayDuration to prevent users from inadvertently specifying huge, delay values which could result in updates getting stuck.<br>Checks have been added in service creation/updation, app upgrade, application & fabric upgrade overrides to prevent users from configuring anything more than the max config value.<br>The max value is a cluster configuration, defaulting to 30mins, so an InstanceCloseDelay can't be specified more than this value. Users can modify this value to anything as they require in the cluster config.<br>For already misconfigured services, the delay would be ceiled to the max config value.<br>**Documentation Reference**: For more about it at <ul><li>[Service Fabric application upgrade: Advanced topics](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-application-upgrade-advanced)</li></ul>**Breaking Change**: This would be a **breaking change** if customers are already configuring more than 30 mins delay.<br>**Workaround**: They would need to define a higher value in the cluster config. |
+| 16. | **Feature** | Added ability to enumerate actor reminders | **Brief Description**: Exposed an API to list all the reminders in the actor service.GetReminderAsync() API has been added to IActorService to enable users to enumerate reminders. <br>**Solution**: Actor customers now has the ability to enumerate the reminders in their service. If a null value is passed to the actorId filter, then the Api returns list of all the reminders. If a valid actor id is passed, then all the reminders corresponding to the actor id input are returned.<table><tr><td><pre lang="csharp">Task<ReminderPagedResult<KeyValuePair<ActorId, List<ActorReminderState>>>> GetRemindersAsync(<br>    ActorId actorId,<br>    ContinuationToken continuationToken,<br>    CancellationToken cancellationToken);</pre></td></tr><tr><td></table>The page size of the result can be controlled by setting the PageSize static field of the ReminderPagedResult inside the actor service.<table><tr><td><pre lang="csharp">ReminderPagedResult<KeyValuePair<ActorId, List<ActorReminderState>>>.SetDefaultPageSize(/*new size*/)</pre></td></tr></table>**Documentation Reference**: Read more about it at -<ul><li>[IActorService.GetReminderAsync(ActorId, ContinuationToken, CancellationToken) Method](https://docs.microsoft.com/en-us/dotnet/api/microsoft.servicefabric.actors.iactorservice.getreminderasync)</li></ul> |
```

Comment:
```suggestion
| 16. | **Feature** | Added ability to enumerate actor reminders | **Brief Description**: Exposed an API to list all the reminders in the actor service.GetReminderAsync() API has been added to IActorService to enable users to enumerate reminders. <br>**Solution**: Actor customers now have the ability to enumerate the reminders in their service. If a null value is passed to the actorId filter, then the API returns list of all the reminders. If a valid actor ID is passed, then all the reminders corresponding to the actor ID input are returned.<table><tr><td><pre lang="csharp">Task<ReminderPagedResult<KeyValuePair<ActorId, List<ActorReminderState>>>> GetRemindersAsync(<br>    ActorId actorId,<br>    ContinuationToken continuationToken,<br>    CancellationToken cancellationToken);</pre></td></tr><tr><td></table>The page size of the result can be controlled by setting the PageSize static field of the ReminderPagedResult inside the actor service.<table><tr><td><pre lang="csharp">ReminderPagedResult<KeyValuePair<ActorId, List<ActorReminderState>>>.SetDefaultPageSize(/*new size*/)</pre></td></tr></table>**Documentation Reference**: Read more about it at -<ul><li>[IActorService.GetReminderAsync(ActorId, ContinuationToken, CancellationToken) Method](https://docs.microsoft.com/en-us/dotnet/api/microsoft.servicefabric.actors.iactorservice.getreminderasync)</li></ul> |
```

Reply:
^"Actor customers now has"-->"Actor customers now have", capitalized second instance of API, capitalized two instances of ID